core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2689,6 +2697,105 @@ write_files:
            echo "Finished Loading EFS Persistent Volume"'
 
 {{ end }}
+{{if .Experimental.Kube2IamSupport.Enabled }}
+  - path: /srv/kubernetes/manifests/kube2iam-ds.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: DaemonSet
+      metadata:
+        name: kube2iam
+        namespace: kube-system
+        labels:
+          app: kube2iam
+        annotations:
+          scheduler.alpha.kubernetes.io/critical-pod: ''
+      spec:
+        template:
+          metadata:
+            labels:
+              name: kube2iam
+          spec:
+            updateStrategy:
+              type: RollingUpdate
+            {{- if .Experimental.Plugins.Rbac.Enabled}}
+            serviceAccountName: kube2iam
+            {{- end}}
+            hostNetwork: true
+            containers:
+              - image: jtblin/kube2iam:latest
+                name: kube2iam
+                args:
+                  - "--app-port=8282"
+                  - "--base-role-arn=arn:aws:iam::{{.AWSAccountID}}:role/"
```

Comment:
i would recommend that these flag is removed and people annotate their pods with the full ARN, it works better if you need to assume roles in several AWS accounts and not only the same account of the AWS cluster